tasks/clone_scl_repo.yml

```diff
@@ -12,4 +12,4 @@
     pretty_print: yes
     add_children:
       - testcase:
-          name: "{{ stuff.scl_url }}"
```

Comment:
I think you will want to leave this unchanged. By changing this, the test case name will look like the following:

```
<?xml version='1.0' encoding='UTF-8'?>
<testsuite name="rhscl-tests" errors="0" failures="0" skipped="0" tests="1">
  <testcase name="/tmp/rhscl_openshift_dir/rhscl-testing-results.xml">
    <system-out>OK (unknown bytes)</system-out>
  </testcase>
</testsuite>
```

Without changing it, the test case name will reflect the actual test being run. Which will make each test case unqiue in the file.

```
<?xml version='1.0' encoding='UTF-8'?>
<testsuite name="rhscl-tests" errors="0" failures="0" skipped="0" tests="1">
  <testcase name="s2i-python-container">
    <system-out>OK (unknown bytes)</system-out>
  </testcase>
</testsuite>
```

Reply:
The PR itself is still in WIP, but good catch. I am going to change it to `testcase name="rhel8-s2i-perl-526-container`. Fix will be available soon.